crates/proto/src/h2/h2_client_stream.rs

```diff
@@ -8,13 +8,15 @@
 use alloc::boxed::Box;
 use alloc::string::String;
 use alloc::sync::Arc;
+use alloc::vec::Vec;
 use core::fmt::{self, Display};
 use core::future::Future;
 use core::net::SocketAddr;
 use core::ops::DerefMut;
 use core::pin::Pin;
 use core::str::FromStr;
 use core::task::{Context, Poll};
+use http::{HeaderName, HeaderValue};
```

Comment:
nit: I think we'd want this `use http::*` to be grouped with the pre-existing external crate `use http::` on L29.